packages/algob/src/builtin-tasks/run.ts

```diff
@@ -34,30 +33,25 @@ function filterNonExistent (scripts: string[]): string[] {
 }
 
 export function mkDeployer (
```

Comment:
let's move this function somewhere to `src/internal`

Reply:
Alright

Reply:
Moving to `src/internal/util/deployer.ts`

Reply:
like in other comment. Please don't move it to `util`. Sorry. I should have note about it earlier.

Reply:
Alright